cmd/erasure-sets.go

```diff
@@ -882,7 +883,17 @@ func (s *erasureSets) parentDirIsObject(ctx context.Context, bucket, parent stri
 	if parent == "." {
 		return false
 	}
-	return s.getHashedSet(parent).parentDirIsObject(ctx, bucket, parent)
+	var isParentDirObject func(string) bool
```

Comment:
You can remove isParentDirObject and call erasureSets.parentDirIsObject instead,

Reply:
Good idea..

Reply:
keeping the code as is - it's more readable the way it is.